bottlecap/src/lifecycle/invocation/processor.rs

```diff
@@ -255,7 +347,7 @@ impl Processor {
         // Set the report log metrics
         self.enhanced_metrics.set_report_log_metrics(&metrics);
 
-        if let Some(context) = self.context_buffer.remove(request_id) {
+        if let Some(context) = self.context_buffer.get(request_id) {
```

Comment:
Doing a `get` since I wanna make sure that only when the buffer is empty, we can calculate the init tags correctly